### PR TITLE
Added README.md for Arithmethic_Progression

### LINKED ISSUE #1305: Add README.md for Arithmetic Progression
I'm GSSoC Participant.
I would like to add README.md for Arithmetic Progression

### PR DESCRIPTION
**Pull request by gauravburjwal**
Fixes #1305

#### Checklist:
- [ ] 4 space indentation.
- [ ] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [ ] Sample Input / Output is added at the end of file.
- [ ] Logic Documentation (Comments).
- [x ] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added README.md for Arithmetic_Progression
 

#### Languages Used:
- Markdown
 

#### Files Added:
- README.md
 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Arithmetic_Progression/README.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+> 🚀 [Compile Online](https://repl.it/@gauravburjwal/Arithmetic-Progression-in-Go) 🚀
+
+
+
+
+
+
+
+
+
```
Comment (marwahmanbir) [inline Arithmetic_Progression/README.md]: Remove unnecessarily added extra lines.

File: Arithmetic_Progression/README.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+For instance, the sequence _1,4,7,10,13,16....._ is an arithmetic progression with _common difference_ of _3_.
+
+( 16-13 = 13-10 = 10-7 = 7-4 = 4-1 = 3 )
+
+
+
+![Illustration: Arithmetic Progression](<https://cdncontribute.geeksforgeeks.org/wp-content/uploads/4-min-1.png>)
+
+​                           ***Illustration of Arithmetic Progression. [Image Courtesy: GeeksforGeeks]***
```
[the remark this answers is not in the document]
Comment (gauravburjwal) [reply Arithmetic_Progression/README.md]: Actually, I'm trying to put this text in the middle but it didn't work. I am using typora editor for my markdown work. Anyways I have made the requested changes.

**Comment by gauravburjwal** [comment]
@marwahmanbir Kindly review again.

File: Arithmetic_Progression/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+# Arithmetic Progression (A.P)
```
Comment (marwahmanbir) [inline Arithmetic_Progression/README.md]: Do add a single line after all the headings. I never told you to remove those.

File: Arithmetic_Progression/README.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+## Properties of Arithmetic Progressions
+
+**Increasing/Decreasing Sequence:**
+
+- If the common difference is positive, i.e. *d>0*  then the arithmetic progression is an *increasing* sequence and satisfies the condition *a<sub>n-1</sub> > a<sub>n</sub>* :
+
+  ​                                                                                 *a<sub>1</sub> < a<sub>2</sub> < a<sub>3</sub> < ...*
+
+  For example, the arithmetic progression with initial term *2* and common difference *3* i.e. *2,5,8,11,14,....,*  is an increasing sequence.
```
[the remark this answers is not in the document]
Comment (marwahmanbir) [reply Arithmetic_Progression/README.md]: > Remove space before `For`

This is also not resolved yet.

File: Arithmetic_Progression/README.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+- If the common difference is negative, i.e.  *d<0* then the arithmetic progression is a *decreasing* sequence and satisfies the condition *a<sub>n-1</sub> > a<sub>n</sub>* :
+
+  ​                                                                                 *a<sub>1</sub> > a<sub>2</sub> > a<sub>3</sub> > ...*
+
+  As an example, the arithmetic progression with initial term *1* and common difference *-3*  i.e.*1,-2,-5,-11,...,*  is a decreasing sequence.
```
[the remark this answers is not in the document]
Comment (marwahmanbir) [reply Arithmetic_Progression/README.md]: > Remove space before `As`

This is also not resolved yet.

File: Arithmetic_Progression/README.md (hunk 3 above)
Comment (gauravburjwal) [reply Arithmetic_Progression/README.md]: Ohh. I mistaken it.

**Comment by gauravburjwal** [comment]
@marwahmanbir All the changes are made. Kindly review and sorry for the inconvenience.

**Comment by marwahmanbir** [comment]
Good work.

**Comment by gauravburjwal** [comment]
Thanks. Kindly approve new issue.

**Pull request merged**